release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -85,11 +85,135 @@ The {product-title} installation program for Google Cloud Platform (GCP) now cre
 
 [id="ocp-4-8-web-console"]
 === Web console
-
 [id="ocp-4-8-custom-console-routes-use-custom-domains-cluster-api"]
 ==== Custom console routes now use the new CustomDomains cluster API
 
 For `console` and `downloads` routes, custom routes functionality is now implemented to use the new `ingress` config route configuration API `spec.componentRoutes`. The Console Operator config already contained custom route customization, but for the `console` route only. The route configuration via `console-operator` config is being deprecated. Therefore, if the `console` custom route is set up in both the `ingress` config and `console-operator` config, then the new `ingress` config custom route configuration takes precedent.
+For `console` and `downloads` routes, custom routes functionally is now implemented to use the new `ingress` config route configuration API. The Console Operator already contained custom route customization, but for the `console` route only. Now, if the `console` custom route is set up in both the `ingress` config and `console-operator` config, then the new `ingress` config custom route configuration takes precedent, because the route configuration via `console-operator` config is being deprecated.
```

Comment:
This paragraph is a duplicate of the preceding paragraph; you can delete this one. Thank you!

Reply:
@ktania46 FYI

Reply:
Handling post-merge via https://github.com/openshift/openshift-docs/pull/33633.